gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/PipelineController.groovy

```diff
@@ -308,4 +309,12 @@ class PipelineController {
       this.additionalAttributes = additionalAttributes
     }
   }
+
+  @ResponseStatus(HttpStatus.UNPROCESSABLE_ENTITY)
+  @InheritConstructors
+  class PipelineExecutionException extends RuntimeException {
```

Comment:
could you have this extend [InvalidRequestException](https://github.com/spinnaker/kork/blob/master/kork-exceptions/src/main/java/com/netflix/spinnaker/kork/web/exceptions/InvalidRequestException.java)

Reply:
@cfieber updated; sorry for the delay.

Reply:
@cfieber mind taking a look? I think we've addressed the type concern.